src/DependencyInjection/PrestaSitemapExtension.php

```diff
@@ -40,6 +42,19 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter($this->getAlias() . '.defaults', $config['defaults']);
         $container->setParameter($this->getAlias() . '.default_section', (string)$config['default_section']);
 
+        if ($container->hasParameter('event_dispatcher.event_aliases')) {
```

Comment:
Maybe I'm missing something, but I was pretty sure that we should never rely on others logic at extension time and use compiler pass instead...
Or maybe you just don't care here because there is a if, and every code in all the Symfony ecosystem should perform this array_merge trick (even Symfony) ?

Reply:
I'm exactly not sure where this part should be. I just found out today that this parameter can be set for this case and I though that at last I could get rid of using event name with this bundle :D

If compiler pass is the better place I can add it there. In Symfony source code I didn't found any usages, so this is added so external bundles could add mappings.